### ISSUE TITLE
Check Validation Loss

### ISSUE THREAD

**Issue by puelon** [SadmanSakib93/Stratified-k-fold-cross-validation-Image-classification-keras#4]
Hello, I was wondering how do I know if the model is overfitting? The accuracy and test accuracy is great and all, but how would I verify the validation loss/history without ruining the great accuracy?

**Comment by SadmanSakib93** [comment]
If you check the code output again please, you will find that both the validation accuracy and loss for each fold are computed.
